src/api/api.js

```diff
@@ -218,18 +244,25 @@ class Api {
       },
     };
     const resObjKey = "user";
-    return this.sendPromise(requestData, resObjKey);
+    return this.#sendPromise(requestData, resObjKey);
+  }
+
+  async userLogin(data) {
+    const { login, password, accessToken } = data || {};
+
+    const requestData = { device_id: this.deviceId };
+    if (login && password) {
+      requestData.login = login;
+      requestData.password = password;
+    } else if (accessToken) {
+      requestData.access_token = accessToken;
```

Comment:
no need to pass `access_token ` in body, it's already passed as header in `#sendHttpPromise`